lisa/tools/gcc.py

```diff
@@ -22,6 +22,14 @@ def compile(self, filename: str, output_name: str = "") -> None:
         else:
             self.run(filename)
 
+    def version(self) -> int:
+        output = self.run(
+            "-dumpversion",
+            expected_exit_code=0,
+            expected_exit_code_failure_message="fail to get gcc dumpversion",
+        )
+        return int(output.stdout.split(".")[0])
```

Comment:
how about return the "VersionInfo" type? Maybe the minor version is needed in future.

Reply:
update